frontend/cypress/integration/all.spec.js

```diff
@@ -27,6 +27,10 @@ const navigateTo = (section, button) => {
     cy.get('button').contains(button).click()
 }
 
+const navigateToMenuItem = (item) => {
+  cy.get('.ant-layout-sider-children').contains(item).click()
+}
+
```

Comment:
How is this different than `navigateTo` with a single argument?

Reply:
Your navigateTo method requires a button, these are lists on the side, not action buttons. Any suggestions?

Reply:
The second argument is for buttons, the first only targets list items: https://github.com/c3g/freezeman/blob/dbc1ed9d06f6628bf615da4d12436a98a3fd43ca/frontend/cypress/integration/all.spec.js#L25